manifests/0000_26_cloud-controller-manager-operator_02_rbac.yaml

```diff
@@ -115,10 +114,26 @@ metadata:
     include.release.openshift.io/self-managed-high-availability: "true"
     include.release.openshift.io/single-node-developer: "true"
 roleRef:
-  apiGroup: rbac.authorization.k8s.io
   kind: Role
   name: cluster-cloud-controller-manager
 subjects:
   - kind: ServiceAccount
     namespace: openshift-cloud-controller-manager-operator
     name: cluster-cloud-controller-manager
+
+# ---
```

Comment:
Do we need to keep the commented RoleBinding here?

Reply:
Accidentally slipped between commits, will squash it in a second.